search/src/main/kotlin/com/ritense/search/service/SearchFieldService.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.search.service
+
+import com.ritense.search.domain.SearchFields
+import com.ritense.search.repository.SearchFieldRepository
+import org.springframework.http.HttpStatus
+import org.springframework.web.server.ResponseStatusException
+
+class SearchFieldService(
```

Comment:
There is an error when starting the application. Because an other service with this name already exist. Can you rename it to: `SearchFieldV2Service`